scripts/maintenance/migrate_project/src/cli.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+
+import typer
+from db import (
+    db_connection,
+    get_project_and_files_to_migrate,
+    insert_file_meta_data,
+    insert_projects,
+)
+from models import Settings
+from r_clone import assemble_config_file, sync_file
+
+
+def main(config: Path):
+    assert config.exists()  # nosec
+    settings = Settings.load_from_file(config)
+    typer.echo(f"Detected settings:\n{settings.json(indent=2)}\n")
+
+    assert settings.source.s3.endpoint is not None  # nosec
```

Comment:
Why you dont simply add it as required in the settings? I do not understand that